content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -459,6 +459,35 @@ validation and auto-completion.
 ⚠️ Note that using the `checkpoint` field in `dvc.yaml` is not compatible with
 `dvc repro`.
 
+## Standalone plots configurations
+
+The list of plots contains one or more user-defined
+[plots](/doc/command-reference/plots#standalone-plots). Here's an example that
+tells DVC that `auc.json` is viable for visualization:
```

Comment:
```suggestion
## Refusable plots

While [plots](/doc/command-reference/plots#standalone-plots) are ultimately
associated to specific [stages](#stages), you can define one or more of them under a top level
`plots` field in `dvc.yaml` in order to customize them once and reuse across multiple
stages.

The following example tells DVC that output `auc.json` is viable for visualization:
```

Reply:
`Refusable` -> `Reusable`

@jorgeorpinel I'm fine with the descriptions, but I'm struggling to understand how the plots are more "reusable" across multiple stages in the top-level field. They enable covering multiple data sources in one plot or using the same data source in multiple plots (maybe that last part is what you mean?).

Reply:
My understanding here is still not great (didn't have time to read all the motivation in core tickets yet). I thought reusability was the main aspect of this but let's find some other word or go with **Top-level plot definitions**. Or just **Plots** if we're planning to (soft) deprecate stage-level `plots`.

Reply:
"Top-level plot definitions" works for now. Let's not let the naming hold up getting this merged because there's a lot of useful content in it.

Reply:
So I should update the PR and replace `standalone` plots with `top-level definitions`?

Reply:
Yup, that sounds good to me.

Reply:
Done

Reply:
> They enable covering multiple data sources in one plot or using the same data source in multiple plots

> The main point is that they can use data sourced from different files, maybe we should include that?

Yes, this motivation should be very clear in the docs, probably presented as benefits or features of plots definitions (more than as advantages over stage-level plots, since those may become deprecated soonish -- we shouldn't assume users will know about those as a base case anyway).

I see some motivation is presented in **[Top-level plot definitions](https://dvc-org-7086-flexible-p-d0hfxv.herokuapp.com/doc/user-guide/project-structure/pipelines-files#top-level-plot-definitions)** (currently in the cmd ref). I'll comment there more specifically... ⏳